### ISSUE TITLE
S/MIME support

### ISSUE THREAD

**Issue by maicol07** [bkaankose/Wino-Mail#101]
Wino Mail is currently lacking S/Mime support for signing and/or encrypting emails

**Comment by Cinzya** [comment]
Are there any plans to add support for this?

**Comment by bkaankose** [comment]
Yes. This is something I'd like to have as soon as possible. Marking as confirmed to have it prioritized.
